web/js/containers/sidebar/smart-handoff.js

```diff
@@ -580,7 +580,7 @@ class SmartHandoff extends Component {
           <hr />
           {this.renderLayerChoices()}
           <hr />
-          {this.renderCropBox()}
+          {!selectedCollection ? undefined : this.renderCropBox()}
```

Comment:
This works great! Can we move the conditional check to the return statement for the `renderCropBox()` function on line 437?

For example:
`return selectedLayer && selectedCollection && proj.id === 'geographic' && ( ... )`

That way all the logic for determining if this shows or not lives in the same place.

Reply:
Thanks, I've consolidated the conditional check for the cropbox. It still works the same.